packages/blade/src/components/Box/LayoutPrimitivesDocs.tsx

```diff
@@ -0,0 +1,462 @@
+      </Section>
+      <Section id="box-usage">
+        <Title size="small" marginBottom="spacing.3">
+          📦 Box Usage
+        </Title>
+        <Text>
+          Box is a primitive Layout component which can be used for creating different responsive
+          layouts in UI. You might have used <Code>Box</Code> in other component libraries as well
+          such as{' '}
+          <Link target="_blank" href="https://chakra-ui.com/docs/components/box">
+            Chakra
+          </Link>
+          . Our Box is similar, except it&apos;s primarily focussed towards layout properties and
+          works on all platforms.
+        </Text>
+        <Text marginY="spacing.4">The simplest Box usage would look something like this-</Text>
+        <SandboxHighlighter>
+          {`
+              <Box>Hello</Box>
+              // On Web          -> <div>Hello</div> 
+              // On React Native -> <View>Hello</View>
+            `}
+        </SandboxHighlighter>
+        <Section id="adding-margin-and-padding">
+          <Heading size="large" marginBottom="spacing.3">
+            ↔️ Adding Margin and Padding
+          </Heading>
+          <Text>Uncomment the commented code below to see things in action ✨</Text>
+          <Sandbox padding={['spacing.5', 'spacing.0', 'spacing.5']} editorHeight={500}>
+            {`
+              import { Box, Text } from '@razorpay/blade/components'
+
+              function App(): JSX.Element {
+                return (
+                  <>
+                    <Box 
+                      // Uncomment next lines to see padding and margin in action
+                      // padding="spacing.4"
+                      // marginTop="32px"
+                      backgroundColor="surface.background.level2.lowContrast"
+                    >
+                      <Text>Some Text</Text>
+                    </Box>
+                    {/*
+                      <Box
+                        // Uncomment this block to see padding shorthands in action
+                        padding={["spacing.3", "35px"]} // We also support padding and margin shorthands similar to CSS
+                        marginX="spacing.5" // adds horizontal margin
+                      >
+                        <Text>More Text</Text>
+                      </Box>
+                    */}
+                  </>
+                )
+              }
+  
+              export default App;
+              `}
+          </Sandbox>
+          <Text>
+            Blade supports multiple props like <Code>padding</Code>, <Code>paddingX</Code>,{' '}
+            <Code>paddingY</Code>, <Code>paddingTop</Code>, <Code>paddingRight</Code>,{' '}
+            <Code>paddingBottom</Code>, <Code>paddingLeft</Code> + similar props with{' '}
+            <Code>margin</Code>. And these props can have values such as <Code>spacing.3</Code> (Our
+            tokens), <Code>132px</Code> (absolute values), <Code>auto</Code>
+          </Text>
```

Comment:
Make it the next bullet point?
```suggestion
            <Code>margin</Code>.
            - These props can have values such as <Code>spacing.3</Code> (Blade
            Tokens), <Code>132px</Code> (Absolute Values), and <Code>auto</Code>
          </Text>
```

Reply:
done in upcoming commit